force-app/main/default/staticresources/pmm/attendancePrintOverride.css

```diff
@@ -36,3 +36,10 @@
 div.slds-form-element__control > .slds-form-element__static {
     border: 0px;
 }
+
+.attendance-row label {
```

Comment:
Since we're not able to apply slds-truncate directly to the input-field labels, we can use our existing staticresource (already being pulled into the component) to apply this CSS manually.  We've scoped this to a top-level class called attendance-row.